### PR TITLE
Inclusion request vorbis/1.3.5@bincrafters/stable

### PR DESCRIPTION
**Pull request by pvicente**
references bincrafters/community#186

CC: @solvingj @uilianries

### CODE REVIEW

File: .travis/install.sh [hunk 1]
```diff
@@ -19,7 +19,5 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     pyenv activate conan
 fi
 
-pip install conan --upgrade
-pip install conan_package_tools
-
+pip install conan_package_tools bincrafters_package_tools
```
Comment (SSE4) [inline .travis/install.sh]: this won't upgrade conan installation. is this intended?

File: test_package/conanfile.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
-from conans import ConanFile, CMake, tools, RunEnvironment
+from __future__ import print_function
```
Comment (SSE4) [inline test_package/conanfile.py]: I think it's better to use self.warn instead of print?

**Comment by pvicente** [comment]
@SSE4 see https://github.com/bincrafters/conan-vorbis/pull/4 I can't edit on top of this PR (no permissions in this repo).

File: .travis/install.sh (hunk 1 above)
Comment (pvicente) [reply .travis/install.sh]: good catch!!! fixing it!!!

File: test_package/conanfile.py (hunk 2 above)
Comment (pvicente) [reply test_package/conanfile.py]: Done!!!

**Comment by uilianries** [comment]
@pvicente I just merged the PR #4, however I'm not able to move this PR into testing/1.3.5. Maybe you need to rebase before to continue.

**Comment by pvicente** [comment]
Hi @uilianries, this PR is coming from testing/1.3.5 and it's pointing to stable/1.3.5 so you only need to accept it, it's up to date with PR https://github.com/bincrafters/conan-vorbis/pull/4 changes.

CC: @SSE4

**Review by uilianries — APPROVED** [review]
LGTM

**Pull request merged**